substrate/client/utils/src/mpsc.rs

```diff
@@ -55,7 +55,7 @@ impl<T> Clone for TracingUnboundedSender<T> {
 	fn clone(&self) -> Self {
 		Self {
 			inner: self.inner.clone(),
-			name: self.name,
+			name: self.name.clone(),
```

Comment:
from a quick search it looks like `TracingUnboundedSender` isn't being cloned anywhere in the hotpath, but leaving this here in case someone knows different

Reply:
No more cloning.